Universal/apic.md

```diff
@@ -0,0 +1,71 @@
+# Patching the APIC table
+
+An odd issue some HEDT users may experience is a kernel panic relating to more cores defined in APIC than actually present:
+
+```
+panic(cpu 0 caller 0x......): "x86_validate topology() 32 threads but 33 registered from MADT~0/...
+```
+
+The reason for this kernel panic is due to macOS's assumption that the first CPU in ACPI is always enabled, which on servers and HEDT systems may not always true.
```

Comment:
```suggestion
The reason for this kernel panic is due to macOS's assumption that the first CPU in ACPI is always enabled, which on servers and HEDT systems may not be always true.
```

Reply:
The reason for this kernel panic is due to macOS's assumption that the first CPU in ACPI is always enabled, which on servers and HEDT systems may not ~~be~~ always **be** true.

Reply:
Whoops